### DISCUSSION TITLE
App will only run with elevated permissions

### DISCUSSION THREAD

**Discussion by JonPerkinsOldGit** [microsoft/WindowsAppSDK#4023] [Q&A]
I've had a WinAppSdk app out to my customers since Feb 2023, with an update in May. Each time it goes out I update the NuGet references. I've just released my latest version, built with the latest WinApp 1.4 SDK. It's installing and running OK on most computers but one customer with 4 computers (Windows 10, updates are up-to-date) cannot get it to run on any of them, although when I had a look I could actually get it to run as administrator.  The app is not doing anything that requires elevated permissions, as evidenced by it running OK everywhere else, and there is no anti-ransomware program running anywhere that I can see. Can anybody offer any opinion as to why a WinAppSdk program would only run with elevated permissions? 

Thanks in advance.

**Comment by Yewzir** [comment]
@JonPerkinsOldGit 

Could it actually be as simple as turning on Developer Mode?

![image](https://github.com/user-attachments/assets/e578f90e-3bce-480a-bcdc-3cbc74caae79)

It's just a first thought, because I think I found myself in a similar situation, years ago. Developer Mode resolved all issues back then.

Kind regards
Yewzir
